docs/bash_to_xsh.rst

```diff
@@ -137,4 +138,12 @@ there is `bashisms xontrib <https://xon.sh/xontribs.html#bashisms>`_:
     echo
     >>> !!
     echo
+    >>> xontrib load sh
+    >>> $AND = 'a' + 'nd'
+    >>> # Run bash script:
+    >>> ! for i in {1..3}
+          do
+             echo -n "$AND $i "
+          done
```

Comment:
This block should go in the description for the xontrib.  Not in the translation guide here

Reply:
removed